DEVELOPMENT.md

```diff
@@ -66,3 +66,15 @@ To do so, you need to have the following environment variables set:
 - AWS_REGION: The region you're building in
 - DOCKER_USERNAME: A username for docker to pull docker images
 - DOCKER_PASSWORD: The password for the username
+
+## Running Tests
+
+We are using playwright to test Zeno full-stack.
+To spin up a dummy database, server, and frontend, there is a separate test `docker compose` file that can be used to run tests.
+You should stop your dev docker containers to prevent port clashes.
+
+You will need to slightly change your `.env` file.
+Use `PUBLIC_BACKEND_ENDPOINT = "http://zeno-backend-test:8000"` for the public backend endpoint, and set the `HUB_USERNAME` and `HUB_PASSWORD` for the test account.
```

Comment:
Is there a way to avoid this? like have a different .env.test file we read instead?

Reply:
Tried for a bit but gave up since we're also using the SvelteKit native functionality of accessing variables in `.env`. Maybe we can find a way but probably for a future PR?

Reply:
This module provides access to runtime environment variables, as defined by the platform you're running on. For example if you're using [adapter-node](https://github.com/sveltejs/kit/tree/master/packages/adapter-node) (or running [vite preview](https://kit.svelte.dev/docs/cli)), this is equivalent to process.env. This module only includes variables that do not begin with [config.kit.env.publicPrefix](https://kit.svelte.dev/docs/configuration#env) and do start with [config.kit.env.privatePrefix](https://kit.svelte.dev/docs/configuration#env) (if configured).

I think it would work with preview if we set the env files variables in the docker file?

Reply:
Where is this from?